driver/web/adapter.go

```diff
@@ -32,6 +32,8 @@ import (
 	"github.com/gorilla/mux"
 	"gopkg.in/go-playground/validator.v9"
 
+	"github.com/casbin/casbin"
+	"github.com/gorilla/mux"
```

Comment:
The build fails with the following error:
# health/driver/web
.../health-building-block/driver/web/adapter.go:35:2: casbin redeclared as imported package name
	previous declaration at .../health-building-block/driver/web/adapter.go:31:2
.../health-building-block/driver/web/adapter.go:36:2: mux redeclared as imported package name
	previous declaration at .../health-building-block/driver/web/adapter.go:32:2

Reply:
This is not fixed yet.